recipes-firmware/firmware/firmware-qcom-boot-common.inc

```diff
@@ -0,0 +1,16 @@
+# Install NHLOS boot binaries in DEPLOY_DIR
```

Comment:
Can you move both files under 'recipes-bsp/firmware/' instead? recipes-firmware/firmware is not really that common.